magento/magento2
# Fix for communication.xml Handlers merging processs

### Description (*)
This changes fixes the "handler" merging process into communication.xml files which currently overrides all the handlers with the last one created, ignoring it's different names as expected. Issue found during custom handler development for MCOM module where there was already a handler for a message.

### Manual testing scenarios (*)
1. Please see https://github.com/magento/magento2/pull/28926#issuecomment-726669211

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29528: Fix for communication.xml Handlers merging processs
